### ISSUE TITLE
[BUG] Dynamic OPML incorrectly declared empty & unpopulated

### ISSUE THREAD

**Issue by jphastings** [FreshRSS/FreshRSS#5727]
**Describe the bug**
I have a (manually created, but [valid](http://validator.opml.org/)) [OPML file](https://www.byjp.me/reading-lists/curiosity/feeds.opml) listing a few RSS feeds. When I add it as a Dynamic OPML feed to FreshRSS, all the feeds listed in the dynamic OPML are declared as "empty".

This seems incorrect because:
- I can one of the RSS feeds _from the FreshRSS UI_, paste into FreshRSS (or my local RSS feed reader) and get content 
- The feed is also valid according to an RSS validator. (See [this validator](https://validator.w3.org/feed/check.cgi?url=https://webcurios.co.uk/feed/))

**To Reproduce**
1. Add this Dynamic OPML file in a new category (https://www.byjp.me/reading-lists/curiosity/feeds.opml or https://www.byjp.me/reading-lists/web-comics/feeds.opml are the ones I have created and used)
2. View feeds in newly created category

**Expected behavior**
The RSS feeds listed are (dynamically) added to the category, and their feed items are listed in FreshRSS.

**Screenshots**
<img width="335" alt="Screenshot 2023-10-24 at 08 32 43" src="https://github.com/FreshRSS/FreshRSS/assets/42999/e578cac3-5cec-499b-9288-1b0a13389286">

<img width="703" alt="Screenshot 2023-10-24 at 08 30 33" src="https://github.com/FreshRSS/FreshRSS/assets/42999/0be015ae-b6d6-4ea5-9b3a-bf6b78b2f724">

<img width="849" alt="Screenshot 2023-10-24 at 08 39 09" src="https://github.com/FreshRSS/FreshRSS/assets/42999/fa890695-d1d4-4e54-9ca5-8ab418cb2073">

**Environment information (please complete the following information):**
- Device: Radxa Rock Pi 4
- OS: Debian Bookworm (Yunohost)
- Browser: [Arc](https://arc.net)
- FreshRSS version: 1.21.0 (Yunohost latest)
- Database version: (Uncertain!)
- PHP version: (Uncertain!)
- Installation type: Yunohost

(And thank you for all your hard work! FreshRSS is awesome!)

**Comment by Alkarex** [comment]
Hello,
Please try to refresh your feeds, and they should get some content

**Comment by jphastings** [comment]
Ahh, thank you so much @Alkarex ! I pressed the "Refresh OPML" button, and assumed it would _also_ refresh the feeds! Having waited a while (without adding and removing new dynamic feeds) was enough to populate the feeds, and pressing "update your feeds" (twice, interestingly) did the same for new dynamic OPML feeds I added.

This is definitely a little confusing seeing an error like that, but there's certainly no _bug_ here, so I'll close this ticket. Thank you!
